### ISSUE TITLE
Underlords hangs with empty window

### ISSUE THREAD

**Issue by ThorodanBrom** [ValveSoftware/Dota-Underlords#32]
#### Your system information

* System information from steam (`Steam` -> `Help` -> `System Information`) in a [gist](https://gist.github.com/ThorodanBrom/1aa1cc0e8ce28a3d5df8bd3e0e87902e): 
* Have you checked for system updates?: [Yes]
* Are you using the latest stable video driver available for your system? [Yes]
* Have you verified the game files?: [Yes]

#### Please describe your issue in as much detail as possible:
Up till the last update Underlords worked perfectly after I update+reboot my system. For this version however, multiple reboots have not done anything

The terminal output when running with `./underlords.sh -vulkan_disable_steam_shader_cache` is in this [gist](https://gist.github.com/ThorodanBrom/2f5c19607c76c1c05e73bb225352d74b). I hope this helps. After printing all this, it hangs.

#### Steps for reproducing this issue:

1. Run Underlords directly from Steam or `./underlords.sh -vulkan_disable_steam_shader_cache`

Thanks for all the work you put into this game!

**Comment by kisak-valve** [comment]
Hello @ThorodanBrom, what version of mesa do you have installed on your system? Intel/Haswell has experimental vulkan support, which requires mesa 19.1.0 or newer for Dota Underlords and your nvidia 710M is a Fermi generation card, which does not support for vulkan.

Possible duplicate of #31.

**Comment by ThorodanBrom** [comment]
I will have to check and tell you, in a while. But if I'm not wrong, I think I'm playing it on the Intel graphics, not on the Nvidia. (The second gist says Intel in a couple of places, that's how I'm guessing :))

Also, I've been playing since the first release, was it this release that stopped support for older cards?

On 27 Sep 2019 10:30 pm, kisak-valve <notifications@github.com> wrote:

Hello @ThorodanBrom<https://github.com/ThorodanBrom>, what version of mesa do you have installed on your system? Intel/Haswell has experimental vulkan support, which requires mesa 19.1.0 or newer for Dota Underlords and your nvidia 710M is a Fermi generation card, which does not support for vulkan.

Possible duplicate of #31<https://github.com/ValveSoftware/Dota-Underlords/issues/31>.

**Comment by ThorodanBrom** [comment]
Hello @kisak-valve, thanks for mentioning checking the Mesa version - when I checked it, I realised that my PRIME profile switched to Nvidia. This was after I had updated my system to Linux Mint 19.2 Tina, which updated Mesa from 19.0.2 to 19.0.8. I switched back to Intel Graphics and it worked perfectly

However, you said that it should work only on 19.1.0, I don't know how it worked for me on 19.0.8!

Thanks and cheers!
